### PR TITLE
Add scm section

### PR DESCRIPTION
**Pull request by marcusholl**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -11,6 +11,12 @@
       <slf4j.version>1.7.7</slf4j.version>
     </properties>
 
+    <scm>
+      <connection>scm:git:git://github.com/SAP/devops-cm-client.git</connection>
+      <developerConnection>scm:git:ssh://github.com:SAP/devops-cm-client.git</developerConnection>
```
Comment (1084565) [inline pom.xml]: Why do we need developer connection?

File: pom.xml [hunk 2]
```diff
@@ -11,6 +11,12 @@
...
+      <url>http://github.com/SAP/devops-cm-client/tree/master</url>
```
Comment (1084565) [inline pom.xml]: Why do we need this additional `url` tag?

File: pom.xml (hunk 1 above)
Comment (marcusholl) [reply pom.xml]: The section here is a prerequisite documented [here](http://central.sonatype.org/pages/requirements.html#scm-information)

File: pom.xml (hunk 2 above)
Comment (marcusholl) [reply pom.xml]: The section here is a prerequisite documented here

**Pull request merged**